varsha/java8features/src/main/java/com/yash/com/java8features/Lambda.java

```diff
@@ -0,0 +1,15 @@
+package com.yash.com.java8features;
+
+interface Addable{  
+    int add(int a,int b);  
+}
+
+public class Lambda {
+
+    public static void main(String[] args) {  
+          
+        Addable ad1=(a,b)->(a+b);  
```

Comment:
can we pass doubles, strings and a combination, how does this behave, how do we restrict it to only strings :question: